<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminstartend</class>
 <widget class="QDialog" name="adminstartend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始/结束售票</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>开始售票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="start_train"/>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_start">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结束售票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="end_train"/>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_end">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="text">
      <string>回到主页</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
